<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmScannerWindow</class>
 <widget class="QDialog" name="FilmScannerWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Film scanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/window/film-scanner</normaloff>:/window/film-scanner</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="hl1">
     <item>
      <widget class="QLabel" name="labelDirectory">
       <property name="text">
        <string>Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eDirectory">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bSelectDirectory">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cSearchInSubdirs">
     <property name="text">
      <string>Search in subdirectories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl2" stretch="0,1,0">
     <item>
      <spacer name="hs1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bScan">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hs2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFounded">
     <property name="title">
      <string>Founded</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="twFounded">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <column>
         <property name="text">
          <string>Filename</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl3">
        <item>
         <spacer name="hs3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bSelectAll">
          <property name="text">
           <string>Select all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bUnselectAll">
          <property name="text">
           <string>Unselect all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bInvertSelection">
          <property name="text">
           <string>Invert selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="l3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelTotalFounded">
       <property name="text">
        <string>Total founded:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTotalFounded">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="l1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSelected">
       <property name="text">
        <string>Selected:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lSelected">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="l2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilmScannerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
